pkg/engine/vulnerability_builder.go

```diff
@@ -107,7 +108,7 @@ var DefaultVulnerabilityBuilder = func(ctx *QueryContext, tracker Tracker, v int
 			linesVulne = detectLine(&file, searchKey, &logWithFields, tracker.GetOutputLines())
 		}
 	} else {
-		logWithFields.Error().Msg("saving result. failed to detect line")
+		logWithFields.Error().Msg("Saving result. failed to detect line")
```

Comment:
Shouldn't error messages always start with lower case?

Reply:
only if it's the error string... e.g: errors.New or errors.Wrap